ftpslib/ftps_zephir_send

```diff
@@ -14,6 +11,11 @@ elif [ "$#" -eq "2" ]; then
   outfile=$2
 else
   echo "usage: $0 infile [outfile]"
+  exit 1
+fi
+
+if [[ -v NOSEND ]]; then
```

Comment:
In particular this construct (check if NOSEND env var is defined) is supported in bash but not in dash